### ISSUE TITLE
Specify font name instead of font path on CLI

### ISSUE THREAD

**Issue by xuhdev** [amueller/word_cloud#346]
It is more common for people to specify a font name instead of a font path when they need to specify a font. It would be awesome if word_cloud can support such a feature. Example:

    wordcloud_cli.py --fontname "Droid Sans" --text mytext.txt --imagefile mycloud.png

**Comment by amueller** [comment]
It can't without adding some heavy dependency.

**Comment by xuhdev** [comment]
I think we can at least leave this open, since dependencies can be made optional. For example, [this package](https://pypi.python.org/pypi/Nikola) depends on a lot more if installed with the option `full`.

**Comment by amueller** [comment]
Yeah we could add an optional dependency and then do something like #25.
